daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

```diff
@@ -88,6 +88,55 @@ You can store a record in Redis using the `create` operation. This sets a key to
 
 An HTTP 204 (No Content) and empty body is returned if successful.
 
+- `get`
+
+You can get a record in Redis using the `get` operation. This get a key where you set.
+
+### Request
+
+```json
+{
+  "operation": "get",
+  "metadata": {
+    "key": "key1"
+  },
+  "data": {
+  }
+}
+```
+
+### Response
+
+```json
+{
+  "data": {
+    "Hello": "World",
+    "Lorem": "Ipsum"
+  }
+}
+```
+
+- `delete`
+
+You can delete multiple records in Redis using the `delete` operation. There is at least one key in metadata, {key: xxx}, if there are more than one, append key1, key2...
```

Comment:
@wxbty 
in the code PR there is no option for deleting multiple keys at the same time ... I think this whole section on `delete` needs to be changed ... 
cc @ItalyPaleAle

Reply:
Because of @ItalyPaleAle's suggestion, the multi-key operation was removed. doc is not synchronized in time